packages/qwik/src/core/v2/client/vnode-diff.ts

```diff
@@ -38,7 +38,7 @@ import {
 import { addPrefixForScopedStyleIdsString, isClassAttr } from '../shared/scoped-styles';
 import type { QElement2, fixMeAny } from '../shared/types';
 import { DEBUG_TYPE, VirtualType } from '../shared/types';
-import type { SsrAttrs } from '../ssr/ssr-types';
+import type { SsrAttrKey, SsrAttrs } from '../ssr/ssr-types';
```

Comment:
We should clean this up. `vnode_diff` is a client only and should not have any references to SSR. We may need to create new types.